src/ServiceManagement/Services/Commands.Utilities/Websites/Services/WebEntities/SiteConfig.cs

```diff
@@ -63,7 +63,7 @@ public interface ISiteConfig
 
         bool? RemoteDebuggingEnabled { get; set; }
 
-        RemoteDebuggingVersion RemoteDebuggingVersion { get; set; }
```

Comment:
It has string constants with the known values now, it's not used elsewhere or compilation would have failed.